js/client-battle.js

```diff
@@ -571,13 +571,14 @@
 				if (this.choice && this.choice.waiting && !this.finalDecision) {
 					buf += '<button name="undoChoice">Cancel</button>';
 				}
-				buf += '</p>';
+				buf += '</p><p class="timer">';
```

Comment:
Maybe we could add options to the main control screen? http://i.imgur.com/d8jHTXv.png